Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Nightborne Race Trait

## Changelog:
- Added Nightborne race trait
- Added flavorization for nightborne culture. Nightborne Dukes are named "Prince", Kings are "Grand Prince" and Emperors are named "King". Duchies are Principalities, Kingdoms are Grand Principalities.


## Developer changelog:
- Added Nightborne race trait
- Couldn't add race trigger since Nightborne don't yet have portraits
- Took the flavorization from #222


# How to test:
- Create a character with the Nightborne culture in the Elven category
- Make sure the trait looks good
- Make sure Nightborne Emperors are called King/Queen and Nightborne Kings/Queens are called Prince/Princess
- Make sure Nightborne kingdoms are called Principalities
